openstack/cronus/templates/cronus/deployment.yaml

```diff
@@ -46,12 +46,27 @@ spec:
           args:
             - "-config=/cronus/config.yaml"
           env:
-          - name: GODEBUG
-            value: {{ .Values.cronus.godebug | quote }}
+            - name: SENTRY_DSN
```

Comment:
This whole deployment file is never used and should probably be deleted. But I think what we have here won't work as it is missing some env variables. In the sts we use envFrom. 

I think we should just delete this file